flybywiresim/a32nx
# fix: Removed simrate decrease limit of 1

## Summary of Changes
The simrate limitation to protect the AP to function correctly also limits to reduce the simrate below 1. 
There is no reason to do this and some users have reported they actually want to do this to be able to slow down to configure the aircraft. 

As it is a standard sim feature we should only deactivate it if there is an actual reason and if it causes actual issues. 

![image](https://user-images.githubusercontent.com/16833201/143214376-6a8ce205-d900-4d9e-a182-cdd205481ad8.png)

## Additional remarks
It might be a good idea to have two new LVARs for MIN and MAX RATE or a single LVAR for turning these limits on and off so that an EFB Setting could be introduced. 


Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
Normal flight starting on gate, runway or air. 
- Activate AP
- Make sure the AP works as expected with simrate <=1
- Make sure the simrate limitation still applies and can be reduced after increasing it. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
